tests/attacks/blocked-flop.mcd

```diff
@@ -0,0 +1,36 @@
+STEPS ( DEPLOY-PRELUDE )
+STEPS ( ATTACK-PRELUDE )
+
+// Attack Sequence
+// ---------------
+
+// Set sump (determines DAI bid size for flop auctions)
+transact ADMIN Vow . file sump 50
+
+// Set dump (determines initial MKR lot size for flop auctions)
+transact ADMIN Vow . file dump 30
+
+// Set tau (flop auction lifetime in seconds)
+transact ADMIN Flop . file tau 3600
```

Comment:
Would these make sense in the DEPLOY-PRELUDE instead?

Reply:
I'm more torn on this one...on one hand, the system would never be "live" without these being initialized, on the other hand I like how having them in the attack file is both self-documenting and robust (i.e. even if someone adds a default setting in the DEPLOY-PRELUDE, no changes to this file are necessary).

To muddy the waters further, `hump` and `bump` are currently set in the `ATTACK-PRELUDE` but should arguably be set in the `DEPLOY-PRELUDE`.

I guess what I'll do is move parameter setting into the `DEPLOY-PRELUDE` and just put a comment in this attack file about what values it's assuming. Slightly more fragile but keeps lines of code down.